modules/prebid/optimization/config/rules-engine-schema.json

```diff
@@ -8,125 +8,123 @@
       "type": "boolean",
       "description": "pending"
     },
+    "generateRulesFromBidderConfig": {
+      "type": "boolean",
+      "description": "pending"
+    },
```

Comment:
Can you remove this for now? The requirements for this have not been formalized yet. We'll introduce this field as part of phase 2 of this module.

Reply:
Sure thing. Removed